contracts/staking/contracts/test/TestStorageLayout.sol

```diff
@@ -73,9 +73,6 @@ contract TestStorageLayout is
             if sub(nextPoolId_slot, slot) { revertIncorrectStorageSlot() }
             slot := add(slot, 1)
 
-            if sub(poolById_slot, slot) { revertIncorrectStorageSlot() }
```

Comment:
Should this have been removed? We still have `poolById`, right?

Reply:
yep, the new `poolById` is the mapping formerly known as `balancesByPoolId`, which lives in `StakingPoolRewardVault`

Reply:
Ah gotcha. Makes sense!